.github/problem-matcher.json

```diff
@@ -0,0 +1,32 @@
+{
+  "problemMatcher": [
+    {
+      "owner": "checkstyle",
+      "pattern": [
+        {
+          "regexp": "^.*\\[(?:(ERROR)|WARN)\\] (.*):\\[(\\d+),(\\d+)\\] \\((.*)\\) (.*)$",
+          "severity": 1,
+          "file": 2,
+          "line": 3,
+          "column": 4,
+          "message": 6,
+          "code": 5
+        }
+      ]
+    },
+    {
+      "owner": "errorprone",
+      "pattern": [
+        {
+          "regexp": "^.*\\[(?:(ERROR)|WARN)\\] (.*):\\[(\\d+),(\\d+)\\] (error: )?\\[(.*)\\] (.*)$",
```

Comment:
the two patterns are similar. why is the first "checkstyle" and the other "errorprone"?

also, maybe we don't need to discern between the two?
the simpler the better

Reply:
The only difference is that Checkstyle has the error code in parenthesis `()` and Error Prone uses brackets `[]`.

Reply:
I can collapse them, any ideas what to put in `owner`?

Reply:
what is `owner` for? you can stick "problem" there?

Reply:
I'm collapsing them and setting `owner` to `maven`.